### PR TITLE
SS-920 no email already exists

### PR DESCRIPTION
**Pull request by churnikov**
## Description

This PR changes signup behavior a bit. From now on we won't show to users that account with email that they provided exists. This should solve minor GDPR issue that we had, that anyone could see whether user with provided email exists or not.

In order to mitigate potential timing attack, we made verification email sending a celery tasks.

## Checklist

_If you're unsure about any of the items below, don't hesitate to ask. We're here to help!
This is simply a reminder of what we are going to look for before merging your code._

- [ x ] This pull request is against **develop** branch (not applicable for hotfixes)
- [ x ] I have included a link to the issue on GitHub or JIRA (if any)
- [ ] I have included migration files (if there are changes to the model classes)
- [ ] I have included, reviewed and executed tests (unit and end2end) to complement my changes
- [ x ] I have updated the related documentation (if necessary)
- [ x ] I have added a reviewer for this pull request
- [ x ] I have added myself as an author for this pull request
- [ ] In the case I have modified settings.py, then I have also updated the studio-settings-configmap.yaml file in serve-charts

## Further comments

Anything else you think we should know before merging your code!

### CODE REVIEW

File: apps/tasks.py [hunk 1]
```diff
@@ -22,6 +22,13 @@
 
 logger = get_logger(__name__)
 
+K8S_STATUS_MAP = {
+    "CrashLoopBackOff": "Error",
+    "Completed": "Retrying...",
+    "ContainerCreating": "Created",
+    "PodInitializing": "Pending",
+}
+
```
Comment (sandstromviktor) [inline apps/tasks.py:31]: I think this was removed in a recent PR. It should be removed at least.

File: common/forms.py [hunk 2]
```diff
@@ -136,15 +140,20 @@ class Meta:
             "username",
         ]
 
+    def is_unique_email(self) -> bool:
```
Comment (sandstromviktor) [inline common/forms.py:143]: Im not a python naming master, but is it not more readable with `email_is_unique`? Is there a reason for putting the `is` first?

File: common/models.py [hunk 3]
```diff
@@ -29,26 +29,7 @@ class EmailVerificationTable(models.Model):
...
-                f" https://{DOMAIN}/verify/?token={self.token}"
-                "\n\n"
-                "SciLifeLab Serve team"
-            ),
-            settings.EMAIL_HOST_USER,
-            [self.user.email],
-            fail_silently=False,
-            html_message=html_message,
-        )
+        send_verification_email(self.user.email, self.token)
```
Comment (sandstromviktor) [inline common/models.py:32]: Maybe we should change the name here or somehow distinguish the two functions? Looks a bit odd imo:)

**Review by sandstromviktor — APPROVED** [review]
I don't know how to test the actual sending of emails, but the change is pretty small, so i'm confident that it works as expected. I've added two comments/questions about naming. Ignore if you don't agree.

File: common/models.py (hunk 3 above)
Comment (churnikov) [reply common/models.py:32]: That's a good idea

**Pull request merged**